client/blocks/edit-gravatar/style.scss

```diff
@@ -115,6 +115,12 @@
 	}
 }
 
+.edit-gravatar.is-uploading {
+	.file-picker {
+		pointer-events: none;
```

Comment:
Using `disabled` doesn't work since the component uses a `div` and not a `button` element. `FilePicker` also doesn't have a way to be disabled.